### ISSUE TITLE
Attention to MXO2 that offer payment services.

### ISSUE THREAD

**Issue by touristcoin** [peatio/peatio#747]
Today one of our affiliated partners has asked for support from this person. Mainly asked for $ 500 with invoice and prepayment. Then we checked out the company he had communicated and was closed for 6 months. Under the conversations. Pay close attention to the scammers and not paid first.
![schermata 2018-05-18 alle 14 03 47](https://user-images.githubusercontent.com/38797338/40233779-978c7208-5aa4-11e8-855c-823052cb7158.png)
![schermata 2018-05-18 alle 14 03 59](https://user-images.githubusercontent.com/38797338/40233780-97b15f1e-5aa4-11e8-9a00-15af44166fb8.png)
![schermata 2018-05-18 alle 14 04 30](https://user-images.githubusercontent.com/38797338/40233782-97cdb1d2-5aa4-11e8-8b4f-eb9ce1178257.png)
![schermata 2018-05-18 alle 14 04 43](https://user-images.githubusercontent.com/38797338/40233783-97ea1df4-5aa4-11e8-9578-cd044101ae07.png)
![schermata 2018-05-18 alle 14 04 58](https://user-images.githubusercontent.com/38797338/40233785-98070310-5aa4-11e8-81d6-8eac49158032.png)
Him is not UK but Indian
Company dissolved : https://beta.companieshouse.gov.uk/company/10244996

**Comment by minersbad** [comment]
ohhhh, thanks for information, we wanted to contact him, but already what we saw is enough

**Comment by touristcoin** [comment]
Yes, make attention to dont pay before. Him before told that name is Sam and company is Maximum oxigen ltd. Him register company online and dont pay tax, the company is dissolved. Can you inform with info@yourcompanyformations.co.uk them have all info. Him is Indian and the name is Singh, Shimbhu

**Comment by minersbad** [comment]
now it's a fashion to cheat. they invent anything
Thanks for information

**Comment by mxo2** [comment]
Hi guys,
Please aware such person and https://touristcoin.net and https://www.listex.io... 

First that person is just blackmailing us for give free services to them ...We have already share everything with him and these are the fact.

1. We didn't call or contact him for payment.
2. He is fake and just wasting our time whole days for asking solutions...we given perfect analysis and facts about the solutions.
3. We ask him to create invoice but he refused and saying pay through the BTC which is illegal.
4. If I ask to create invoice that mean it would be a legal process and mention that it would be from Indian company (see in chat) so how we are illegal.
5. We also told him that our company is registered in UK but we never doing any transaction or Trading there because some trademark issues so how were fraud..?
6. He removed some chat part and we never demand for any payment from him...so make sure before giving any support

We have complete billing and support system...

Finally...He is blackmailing us and we are ready to share whole chat if any one needed...

**Comment by mxo2** [comment]
@minersbad Please ready fact and than share your views..

**Comment by minersbad** [comment]
I know them well and I think it's good to close your mouth.

**Comment by mxo2** [comment]
He is still black mailing us..please check screenshot..

**Comment by mxo2** [comment]
@minersbad this ID is belong to same person..

Because Id  @minersbad is not having any activity but he is replied in this post..so guys these people of https://touristcoin.net and https://www.listex.io... doing scam to just blackmailing and want services free..

**Comment by minersbad** [comment]
where you see it?
here my activity profile from a lot of time
![schermata 2018-05-18 alle 14 47 00](https://user-images.githubusercontent.com/35083727/40235407-725d781e-5aaa-11e8-880c-85ab93121a3e.png)
I think that you are a scam true

**Comment by coinlisting** [comment]
@mxo2 sorry that it means that if someone does not post them they are?

**Comment by coinlisting** [comment]
@mxo2 I honestly saw the conversations on skype, and the facts say otherwise

**Comment by minersbad** [comment]
@coinlisting the important thing is to be careful not to pay these improbable people who offer paid services first

**Comment by coinlisting** [comment]
BTC is illegal? @mxo2 I think you need a doctor

**Comment by minersbad** [comment]
maybe him is illegal :D

**Comment by touristcoin** [comment]
please folks, the important thing that you pay attention to pay this person upfront. Those other people who talk about him.
![schermata 2018-05-18 alle 15 00 52](https://user-images.githubusercontent.com/38797338/40236047-89b735d4-5aac-11e8-8571-fc3c4433bb15.png)
![schermata 2018-05-18 alle 15 01 03](https://user-images.githubusercontent.com/38797338/40236048-89d70de6-5aac-11e8-84af-b7a9777f85a3.png)

**Comment by mxo2** [comment]
these guys saying that they are mobile operator but I think such activity looking very childish...

**Comment by coinlisting** [comment]
@touristcoin quiet, we know who to believe well

**Comment by minersbad** [comment]
:O Apollo IT i know, very seriuos him

**Comment by mxo2** [comment]
I post for apollo IT because his fork is not working ...If posting issue is fake than its amazing...

Other people also agree with that...Still his fork is not stable for us..If it's not working means not working...that means not i am fake..

**Comment by minersbad** [comment]
@mxo2 here we talk that you try to cheat people. Not that the fork works, do you realize it?

**Comment by coinlisting** [comment]
without words. People who are sick and who say that paying BTC is illegal.

**Comment by minersbad** [comment]
@coinlisting it is illegal to give a computer to him.
ahuauhahuuha

**Comment by coinlisting** [comment]
aaabhhaaha

**Comment by touristcoin** [comment]
please folks, the important thing that you pay attention to pay this person upfront. Those other people who talk about him.

**Comment by mxo2** [comment]
I got information and found that these guys developing fraud tourism platform to get bitcoin and making money..This group is not accepting and paying though the banking system..they want btc Transaction..

Please don't invest , support these guys..
https://touristcoin.net and https://www.listex.io

**Comment by touristcoin** [comment]
@coinlisting Hi Mark, dont worry that CEO of ListEX and us have mailed github for this person abuse. now they will take provisions.

**Comment by coinlisting** [comment]
@touristcoin Yes, but I still want to understand why these people are given the computer. Illegal bitcoin. But why do you still trust him

**Comment by minersbad** [comment]
@coinlisting  you are definitely right. @mxo2  It must be ignored

**Comment by touristcoin** [comment]
BTC illegal ahuauhhau. yes @coinlisting i back to work

**Comment by ZahroKa** [comment]
coinlisting , mxo2, salvo, touristcoin and listex are same team of frauders. Be careful people! If you don't want to lose your money, dont use this service. It is a group of unprofessional morons, who do not have any respect for their customers and their opinions.

**Comment by coinlisting** [comment]
you are not credible

![schermata 2018-05-31 alle 17 06 46](https://user-images.githubusercontent.com/39408509/40790686-d5e13ef2-64f5-11e8-83f7-79af8a7e792e.png)
